We do not have hatchling on Solaris; this patch modifies the build
metadata to use setuptools instead.

Not intended for upstream.

--- urllib3-2.5.0/pyproject.toml
+++ urllib3-2.5.0/pyproject.toml
@@ -1,8 +1,8 @@
 # This file is protected via CODEOWNERS
 
 [build-system]
-requires = ["hatchling>=1.6.0,<2", "hatch-vcs>=0.4.0,<0.6.0", "setuptools-scm>=8,<9"]
-build-backend = "hatchling.build"
+requires = ["setuptools", "setuptools-scm>=8,<9"]
+build-backend = "setuptools.build_meta"
 
 [project]
 name = "urllib3"
@@ -115,6 +115,9 @@ docs = [
 "Code" = "https://github.com/urllib3/urllib3"
 "Issue tracker" = "https://github.com/urllib3/urllib3/issues"
 
+[tool.setuptools.dynamic]
+version = {attr = "urllib3._version.__version__"}
+
 [tool.hatch.version]
 source = "vcs"
 [tool.hatch.version.raw-options]
